src/cljs_time/extend.cljs

```diff
@@ -23,20 +24,13 @@
   IEquiv
   (-equiv [o other]
     (and (instance? goog.date.Date other)
-         (.equals o other)))
+         (ct/equal? o other)))
```

Comment:
felt like it's better to lean on the cljs-time library to check for equality rather than the underlying goog.date lib ... for example if cljs-time changed to use some other way to check for equality, then this would remain consistent with that. Not religious about it tho...